### DISCUSSION TITLE
Drift correction not working, Focusrite Clarett 8PRe Thunderbolt, 2CH BlackOut, Logic into Zoom

### DISCUSSION THREAD

**Discussion by sneakinhysteria** [ExistentialAudio/BlackHole#656] [General]
I use this to stream my microphone channel with AU plugins and a sampler instrument channel live into Zoom. I have an AUX channel set up in Logic with the AU plug-ins and I also use this to monitor. So monitoring on Clarett is disabled. 

Logic project is 48kHz, so is the Clarett config, and the Multi Output Device.

Initially, all is perfect. But over a few minutes  (+10mins) a noticeable delay starts building up. This then gets to the point where the setup becomes unusable. This is the same no matter if I make Claret out BlackHole master device. Only unticking the drift correction boxes and re-activating it brings the output back to realtime. But it then builds up again over time. 

I also tried to only use Drift Correction on BlackHole 2CH or Clarett, but no change.

**Comment by devinroth** [comment]
That sucks. I suggest putting in a bug report for macOS. If we annoy them enough maybe they’ll fix it.

Devin

**Reply by sneakinhysteria** [reply]
I tested this also with Clarett and the Zoom Audio driver in a Multi Output Device. Guess what. Same issues. So not specific to BlackHole. So yes, this really seems to be a MacOS. I'm on 12.5.1, Apple silicon. Or could this be a Focusrite issue?

**Comment by devinroth** [comment]
It could be either. If somehow FocusRite isn’t reporting the clock correct it could be an issue but I bet it’s an issue with Apple’s drift correction. In fact, I create my own drift correction algorithm because Apple’s has issues. Sorry it’s not open source though.

Devin
